src/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import cx from '../utils/classnames';
+import { props, t, ReactChildren } from '../utils';
+import FlexView from 'react-flexview';
+
+export type NavBarProps = {
+  /** Left content: It doesn't shrink nor grow */
+  /** Center content. It grows to use all the available space */
+  /** Right content. It doesn't shrink nor grow */
+  /** Max Width. For css `max-width` */
+  content: {
```

Comment:
Are you sure about having all the comments before the object definition?

Reply:
It's because I don't know if I can comment inner properties @giogonzo @FrancescoCioria

Reply:
I think so, anyway you can easily test this "rendering" the component, even from the same file, and see what the VSCode completion popup/tooltip shows you

Reply:
tested, comments in inner props are not supported

Reply:
ok, but this way we don't have a comment for `content`. I would add one

Reply:
No, the comment is the 4 lines paragraph and it's supported by vscode